Scrabble/Play.cs

```diff
@@ -0,0 +1,160 @@
+//-----------------------------------------------------------------------
+
+namespace Scrabble
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using System.Threading.Tasks;
+    using Scrabble.Game_Objects;
+
+    /// <summary>
+    /// This class is responsible for holding data for a play after it has been selected by the user
+    /// throws VaryingParallelListLength exception in initializer if all Lists sent into it are not the same length
+    /// </summary>
+    public class Play
+    {
+        /// <summary>
+        /// a list of letterTiles objects for the current play
+        /// which is a parallel list with xCoordinates and 
+        /// yCoordinates
+        /// </summary>
+        private List<LetterTile> letterTiles;
+
+        /// <summary>
+        /// a list of x coordinates for the current play
+        /// which is a parallel list with yCoordinates and
+        /// letterTiles
+        /// </summary>
+        private List<int> coordinatesX;
+        
+        /// <summary>
+        /// a list of y coordinates for the current play
+        /// which is a parallel list with xCoordinates and
+        /// letterTiles
+        /// </summary>    
+        private List<int> coordinatesY;
+        
+        /// <summary>
+        /// the playerID from an instance of Player which is
+        /// making the current play
+        /// </summary>
+        private int playerID;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Play"/> class.
+        /// </summary>
+        /// <param name="letterTiles">The letterTiles member will be set to this value.</param>
+        /// <param name="coordinatesX">The xCoordinates member will be set to this value.</param>
+        /// <param name="coordinatesY">The yCoordinates member will be set to this value.</param>
+        /// <param name="playerID">The playerID member will be set to this value.</param>
+        public Play(
+            List<LetterTile> letterTiles, 
+            List<int> coordinatesX, 
+            List<int> coordinatesY,
+            int playerID)
+        {
+            if (letterTiles.Count == coordinatesX.Count && coordinatesX.Count == coordinatesY.Count)
+            {
+                this.letterTiles = letterTiles;
+                this.coordinatesX = coordinatesX;
+                this.coordinatesY = coordinatesY;
+                this.playerID = playerID;
+            }
+            else
+            {
+                throw new VaryingParallelListLength("arguments for letterTiles, xCoordinates, and yCoordinates have varying lengths");
+            }
+        }
+
+        /// <summary>
+        /// returns playerID member.
+        /// </summary>
+        /// <returns>The playerID for the player making the move.</returns>
+        public int GetPlayerID()
+        {
+            return this.playerID;
+        }
+
+        /// <summary>
+        /// returns playerID member.
```

Comment:
This is a copy and paste error. The provided summary is not for `GetXCoordinate`, but for `GetPlayerId`.